ProblemsSolution/ProblemsSolution.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq;
+
+namespace ProblemsSolution
+{
+    public class ArrayProblemsSolution
+    {
+        public static int SumArrayEvenNumbers(int[] numbers)
+        {
+            int sum = numbers.Where(number => number % 2 == 0).Aggregate((a, b) => a+b);

```

Comment:
This can be done nicer,

Can you levreag extension (or whatever mecanism c# has) to create an .even method 
Also you should be able to simply use `.Sum`

Reply:
Done a50351dd8b44e6452baf759fd43fda28b05561e0.